src/database/operations.rs

```diff
@@ -20,42 +21,67 @@ pub(crate) async fn get_pull_request(
     repo: &GithubRepoName,
     pr_number: PullRequestNumber,
 ) -> anyhow::Result<Option<PullRequestModel>> {
-    let pull_request = sqlx::query_as!(
-        PullRequestModel,
+    let record = sqlx::query!(
```

Comment:
Remark: it's a bit annoying that we can't use `query_as` anymore when we want to create an attribute from two SQL table field. I haven't investigated this thoroughly, but I wonder if there's a way to still use `query_as` in this case. If you're interested, it would be nice to investigate :)